src/main/java/org/fcrepo/importexport/importer/Importer.java

```diff
@@ -386,40 +444,113 @@ private void importFile(final File f) {
                     logger.info("Importing container {} to {}", f.getAbsolutePath(), destinationUri);
                     response = importContainer(destinationUri, sanitize(model));
                 }
+            }
 
-                if (response == null) {
-                    logger.warn("Failed to import {}", f.getAbsolutePath());
-                } else if (response.getStatusCode() == 401) {
-                    importLogger.error("Error importing {} to {}, 401 Unauthorized", f.getAbsolutePath(),
-                        destinationUri);
-                    throw new AuthenticationRequiredRuntimeException();
-                } else if (response.getStatusCode() > 204 || response.getStatusCode() < 200) {
-                    final String message = "Error while importing " + f.getAbsolutePath() + " ("
-                            + response.getStatusCode() + "): " + IOUtils.toString(response.getBody());
-                    logger.error(message);
-                    importLogger.error("Error importing {} to {}, received {}", f.getAbsolutePath(), destinationUri,
-                        response.getStatusCode());
-                } else {
-                    logger.info("Imported {}: {}", f.getAbsolutePath(), destinationUri);
-                    importLogger.info("import {} to {}", f.getAbsolutePath(), destinationUri);
-                    successCount.incrementAndGet();
+
+            if (response == null) {
+                logger.warn("Failed to import {}", f.getAbsolutePath());
```

Comment:
I realize this is logic that you didn't create, originally, but I wonder why other else clauses use logger.error() and log to `importLogger` and this one doesn't. Maybe it should, since it is an error case that needs to be looked at, right?

Reply:
Yes - I will fix.